slack-api-client/src/main/java/com/slack/api/socket_mode/impl/SocketModeClientJavaWSImpl.java

```diff
@@ -152,8 +152,8 @@ public boolean verifyConnection() {
                         e.getMessage()
                 );
             }
-            if (getLogger().isDebugEnabled()) {
-                getLogger().debug("Failed to receive a pong message");
+            if (getLogger().isTraceEnabled()) {
+                getLogger().trace("Failed to receive a pong message");
             }
```

Comment:
Consider making this a warning.

```suggestion
            getLogger().warn("Failed to receive a pong message");
```